develop-guide/writeblitz/writeblitz.rst

```diff
@@ -550,6 +569,41 @@ Action ``compare`` allows you to verify the values of the saved variables. Below
         - "'%VARIABLES{os}' == 'NX-OS' and '%VARIABLES{date_created}' == '10/22/2019 10:00:00 [10/22/2019 16:57:31]'"
         - " %VARIABLES{bootflash} >= 290000 or '%VARIABLES{bios}' == '07.33'"
 
+
+
+.. note::
+
+    The starting message of an step in etrade logs, is predefined. However, that value can be changed.
+    In your action you can specify your custom message like the example below and change the starting action log message.
+
+.. code-block:: YAML
+
+    - execute:
+        command: show version
+        device: PE1
+        custom_step_message: This is a test to see if a custom_step_message would be applied
```

Comment:
is this the start or end.

change the variable name to custom_start_step_message 

what do you think

Reply:
addressed

Reply:
in the code too right ?

Reply:
Yes